src/kernel/include/shared/cos_types.h

```diff
@@ -414,6 +432,20 @@ struct cos_stack_freelists {
  * COMP_INFO_TMEM_STK_RELINQ != 0.  Change the defines, or change the assembly" */
 /* #endif */
 
+/* Maybe this should be somewhere else... */
```

Comment:
Yeah, we can discuss this. Not sure, but certainly feels like an arch-specific abstraction.